test/distributed/test_dist_link_neighbor_sampler.py

```diff
@@ -435,6 +527,87 @@ def dist_link_neighbor_sampler_temporal_hetero(
         assert out_dist.num_sampled_nodes[k] == out.num_sampled_nodes[k]
 
 
+def dist_link_neighbor_sampler_biased_hetero(
```

Comment:
Could we add `weight_attr` argument to the `dist_link_neighbor_sampler_hetero()` and remove this function?

Reply:
done